plugins/woocommerce/tests/performance/requests/shopper/cart-apply-coupon.js

```diff
@@ -109,43 +112,44 @@ export function cartApplyCoupon() {
 		);
 
 		response = http.post(
-			`${base_url}/?wc-ajax=apply_coupon`,
+			`${ base_url }/?wc-ajax=apply_coupon`,
 			{
-				coupon_code: `${coupon_code}`,
-				security: `${apply_coupon_nonce}`,
+				coupon_code: `${ coupon_code }`,
+				security: `${ apply_coupon_nonce }`,
 			},
 			{
 				headers: requestheaders,
+				tags: { name: 'Shopper - wc-ajax=apply_coupon' },
 			}
 		);
 
-		applyCouponTrend.add(response.timings.duration);
-		check(response, {
-			"is status 200": (r) => r.status === 200,
-			"body contains: 'Coupon code applied successfully'": (response) =>
-				response.body.includes("Coupon code applied successfully"),
-		});
+		check( response, {
+			'is status 200': ( r ) => r.status === 200,
+			"body contains: 'Coupon code applied successfully'": ( response ) =>
+				response.body.includes( 'Coupon code applied successfully' ),
+		} );
 
-		response = http.post(`${base_url}/cart`,
-		{
-        	_wp_http_referer: "%2Fcart",
-			//"cart["+`${item_name}`+"][qty]": "1",
-			coupon_code: "",
-        	"woocommerce-cart-nonce": `${woocommerce_cart_nonce}`,
-		},
-		{
-			headers: requestheaders,
-		}
+		response = http.post(
+			`${ base_url }/cart`,
+			{
+				_wp_http_referer: '%2Fcart',
+				// "cart["+`${item_name}`+"][qty]": "1",
+				coupon_code: '',
+				'woocommerce-cart-nonce': `${ woocommerce_cart_nonce }`,
+			},
+			{
+				headers: requestheaders,
+				tags: { name: 'Shopper - Update Cart' },
+			}
 		);
-		applyCouponCartTrend.add(response.timings.duration);
-		check(response, {
-			"is status 200": (r) => r.status === 200,
-			"body contains: 'woocommerce-remove-coupon' class": (response) =>
-				response.body.includes('class="woocommerce-remove-coupon"'),
-		});
-	});
+		check( response, {
+			'is status 200': ( r ) => r.status === 200,
+			"body contains: 'woocommerce-remove-coupon' class": ( response ) =>
+				response.body.includes( 'class="woocommerce-remove-coupon"' ),
```

Comment:
This assertion is also failing for me.

Reply:
Thanks for flagging, same as the other coupon assert failure it is because a fresh localhost site has no coupons already setup.